tests/parallel_runner.py

```diff
@@ -130,13 +131,15 @@ def updateResult(self, result):
     result.stopTest(self.test)
 
   def startTest(self, test):
-    pass
+    self.startTime = time.perf_counter()
```

Comment:
maybe `start_time`? but i guess it's near python library code that uses `startTest` style...